src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -1020,6 +1024,11 @@ public function valuesArray(): array
         return $ret;
```

Comment:
also i think we might want to change the name of `valuesArray` to just `responses`?  Since it's not actually returning values directly, but rather, an array of objects that may have values in them?

Reply:
@cprice404 this actually is extracting the values from the responses and returning them.

Reply:
See https://github.com/momentohq/client-sdk-php/pull/65#issuecomment-1315780503